README.md

```diff
@@ -74,6 +74,7 @@ vimeo = "{VIMEO_API_TOKEN}"
   # filters = { title = "regex for title here" } # Optional Golang regexp format. If set, then only download episodes with matching titles.
   # opml = true|false # Optional inclusion of the feed in the OPML file (default value: false)
   # clean = { keep_last = 10 } # Keep last 10 episodes (order desc by PubDate)
+  # downloader_opts = [ "-4", "--verbose" ] # Custom command line options for the downloader (youtube-dl)
```

Comment:
Can we name this `youtube_dl` since these are youtube-dl specific options?

Reply:
They are now, because youtube-dl is the only downloader. But what if you add another one? I figured that was the intention behind making `Downloader` an interface.

Reply:
Parameters for youtube-dl will not work for other downloaders, so it doesn't make sense to abstract them.

> But what if you add another one?

There are way too many things can happen in future, you can't predict them all. So let's solve issues as they arrive. For now this will work and I'm ok with it.

Reply:
Who said they are for youtube-dl? When another downloader is introduced there should be another feed parameter to select one. Then `downloader_opts` becomes a universal option that suits any downloader.

Reply:
Otherwise you end up with two choices: rename the parameter and break compatibility, or introduce another parameter named downloader_x_opts which will do exactly the same. In my opinion neither is pretty.

Reply:
I don't have plans to support multiple downloaders, so you're safe to rename it.

Reply:
Feel free to close this one then, I'm quite happy to switch to my fork.